### PR TITLE
Fix for ADJS-92: iOS 8 Smart Banners - Duplicate banner

### PR DESCRIPTION
**Pull request by jnurse**
Removes the smart banner on iOS 8 devices only when it appears above the tag. 

Status: **Opened for visibility**
Reviewers: **@tedtate @donnielrt @scalvert @jansepar**
JIRA: **https://mobify.atlassian.net/browse/ADJS-92**

## Changes
- Remove smart banner meta tag from head when it is above the mobify tag
- Added examples for testing

## How to test-drive this PR
- Checkout this branch
- `grunt serve`
- view /tests/fixtures/smart-banner-below-tag.html on an iOS8 device
- view /tests/fixtures/smart-banner-above-tag.html on an iOS8 device
- A smart banner should appear for both examples
  
## Notes
- Still needs qunit tests.

### CODE REVIEW

File: examples/capturing-smartbanner/above-tag.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (jansepar) [inline examples/capturing-smartbanner/above-tag.html]: Typically the `examples` folder is where all of the code for demonstrations go. Could we move these to the `tests/fixtures` folder?

**Comment by jansepar** [comment]
Btw just going to add myself as a reviewer - I like to have eyes on any changes that happen to capturing since this shit is so cray :)

**Comment by scalvert** [comment]
Good call, Shawn!

**Comment by tedtate** [comment]
@jnurse What happens for iOS 7 browsers in the above/below tag cases?

**Comment by jnurse** [comment]
There's still some weird behaviour with smart banners on iOS 7.x. On iOS 7.1 and 7.0 the below tag example doesn't show the banner. The above tag example works in iOS 7.1 but in 7.0 the banner appears for a second and then disappears.

Looks like that behaviour is consistent with the current version of capturing. See www.womenshealthmag.com for the below tag behaviour, and www.menshealth.com for above the tag.

**Comment by jansepar** [comment]
@jnurse could we add a test to https://github.com/mobify/capturejs/blob/master/tests/capture-tests.js for this? We should mock the `Capture.isIOS8_0` method to return true for the test and then assert that a document with the meta tag does not contain that meta tag in the captured document.

**Comment by jansepar** [comment]
Also, we're going to have to integrate this into adaptivejs. RIght now we're bringing in mobifyjs as a bower component because mobifyjs used to be the repo that contained capture.js - we'll have to point it to this instead.

**Comment by jnurse** [comment]
@jansepar tests have been added!

**Comment by jansepar** [comment]
This looks good @jnurse! Could we make sure to bump the version in the bower.json and package.json, and bump the version on the www site? Once that's done let me know and I can deploy it to the CDN and push to NPM!

**Comment by jansepar** [comment]
:+1:

**Pull request merged**